lib/shared/drive-constraints.js

```diff
@@ -208,3 +208,85 @@ exports.isDriveValid = (drive, image) => {
 exports.isDriveSizeRecommended = (drive, image) => {
   return _.get(drive, [ 'size' ], UNKNOWN_SIZE) >= _.get(image, [ 'recommendedDriveSize' ], UNKNOWN_SIZE);
 };
+
+const feedbackMessages = {
+  NOTREC: 'Not Recommended',
+  TOOSMALL: 'Too Small For Image',
+  LOCKED: 'Locked',
+  SYSTEM: 'System Drive',
+  SOURCE: 'Drive Contains Image'
+};
+const FEEDBACK_ERROR = 'error';
```

Comment:
Actually, can we move this constants inside the `getDriveFeedback()` function? They are very local to that piece of code, and probably don't make much sense as globals for the entire file.